code-challenges.test.js

```diff
@@ -0,0 +1,152 @@
+// ASSESSMENT 3: Coding Practical Questions with Jest
+
+// Please read all questions thoroughly
+// Pseudo coding is REQUIRED
+// If you get stuck, please leave comments to help us understand your thought process
+
+// Use test driven development to complete the following questions
+// Add appropriate dependencies: $ yarn add jest
+
+// Reminder: The test will call your function
+// Run the file with the following command: $ yarn jest
+
+// --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+
+// a) Create a test with expect statements for each of the variables provided.
+describe("fibonacci", () => {})
```

Comment:
Great job setting up your test just make sure the closing }) in this describe line are at the end of the test.